<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoofdScherm</class>
 <widget class="QMainWindow" name="HoofdScherm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verkeersplein Configuratie Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="groenButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Zet groene tijd</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="roodButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zet rode tijd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oranjeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zet oranje tijd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knipperButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zet knippertijd
nachtstand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nachtAanButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zet nachtstand aan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nachtUitButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zet nachtstand uit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pas de tijden aan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aantal seconden</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infoLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Wijzig nachtstand</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>groenButton</zorder>
   <zorder>roodButton</zorder>
   <zorder>oranjeButton</zorder>
   <zorder>knipperButton</zorder>
   <zorder>nachtAanButton</zorder>
   <zorder>nachtUitButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>infoLog</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
